root/etc/e-smith/events/actions/nethserver-directory-tlspolicy

```diff
@@ -33,6 +33,17 @@ replace: olcTLSProtocolMin
 olcTLSProtocolMin: 3.2
 EOF
 
+elif [[ -n ${tlspolicy} ]] && (( $tlspolicy >= 20200510 )); then
+    ldapmodify -c -Y EXTERNAL &>/dev/null <<'EOF' || ((errors ++))
+dn: cn=config
+changetype: modify
+replace: olcTLSCipherSuite
+olcTLSCipherSuite: EECDH:EDH:CAMELLIA:ECDH:RSA:!eNULL:!SSLv2:!RC4:!DES:!EXP:!SEED:!IDEA:!3DES
+-
+replace: olcTLSProtocolMin
+olcTLSProtocolMin: 3.3
+EOF
+
```

Comment:
This should be the first IF branch, so you can simplify the IF checks:

```
if (( $tlspolicy >= 20200510 ))
    ...
elif (( $tlspolicy >= 20180621 ))
    ...
else
    ...
fi

Reply:
done thank